### ISSUE TITLE
Feature request: Open location list and display markers from all buffers

### ISSUE THREAD

**Issue by MartyLake** [kshenoy/vim-signature#149]
Hi,

I was looking for a plugin that traces when I navigate in a new code base. This plugin looks perfect but I can't figure out a way to list all the markers.

Can you help me ?
Best,

**Comment by MartyLake** [comment]
SignatureListBufferMarks does the trick. I don't know why I did not find it before.

**Comment by MartyLake** [comment]
Well, il works only for the current buffer.

**Comment by kshenoy** [comment]
Yes, currently it shows only all marks from the current buffer or all global marks from all buffers. I do like your suggestion but I don't have time to work on it for at least a short while.

For the moment, if you don't have a lot of marks spread across the files, you could use global marks instead.

**Comment by MartyLake** [comment]
Thanks! Do you know how to emulate "m," (mark with next free letter) with global marks ?

**Comment by kshenoy** [comment]
Hi, sorry for the late response. You can try setting `g:SignatureIncludeMarks='ABCDE...Z' so that it will only place global marks.

**Comment by MartyLake** [comment]
Hey, thanks for the patch ! Unfortunately I can't get it working, when I try to "m.", I get an error "No free marks left"

Here is my config 

```
Plugin 'kshenoy/vim-signature' "marks
let g:SignatureIncludeMarks='ABCDEFGHIJKLMNOPQHSTUVWXYZ'
let g:SignatureMarkTextHLDynamic=1
let g:SignatureMarkerTextHLDynamic=1
```
